src/runtime/rpc/network.cpp

```diff
@@ -442,6 +449,21 @@ void rpc_session::start_negotiation()
         }
 
         auth_negotiation();
+    } else {
+        // set negotiation success if auth is disabled
+        complete_negotiation(true);
```

Comment:
In which case to pass `false` to `complete_negotiation `?

Reply:
When negotiation failed, false will passed to `complete_negotiation`

Reply:
Only `true` is passed in this PR, will `false` be passed by another PR?

Reply:
yes, you are right